File: frontend/src/app/models/ClientSortPlaylist.ts
import { MessageBase, MessageType } from './MessageBase';
import { PlaylistSortMode } from './PlaylistSortMode';

export class ClientSortPlaylist extends MessageBase
{
    constructor() {
        super();
    }
    readonly Type : MessageType = MessageType.SortPlaylist;
    PlaylistID : string;
    SortMode : PlaylistSortMode;
    Reverse : boolean;
}

File: BeatOn/ClientModels/ClientSortPlaylist.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using QuestomAssets.BeatSaber;

namespace BeatOn.ClientModels
{
    [Message(MessageType.SortPlaylist)]
    public class ClientSortPlaylist : MessageBase
    {
        public string PlaylistID { get; set; }
        public PlaylistSortMode SortMode { get; set; }
        public bool Reverse { get; set; }
    }
}